common/changes/office-ui-fabric-react/tagitem_2019-02-27-19-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "TagItem: Adding white-space CSS prop which is required for text-overflow\"",
```

Comment:
Also, I think instead of `TagItem` the change is for `TagItemSuggestion`:
```suggestion
      "comment": "TagItemSuggestion: adds `white-space: nowrap` CSS declaration which is required for `text-overflow: ellipsis` to work.",
```